src/napari_imagej/widgets/menu.py

```diff
@@ -305,16 +305,17 @@ def __init__(self, viewer: Viewer):
 
     def _update_settings(self):
         args = {}
-        for k, v in settings.items():
+        default_source = next(s for s in settings.sources if s.default)
+        for k, v in default_source.items():
```

Comment:
`v` is not used here, is it?

Could be changed to:
```
for k, _ in default_source.items():
```

Reply:
Or just `for k in default_source.keys()`, or even `for k in default_source` since Python 3 iterates keys by default on dictionaries.

Reply:
> or even `for k in default_source` since Python 3 iterates keys by default on dictionaries.

Yup, this is great, thanks both for the suggestions. Implemented in 020f10b (along with a couple other changes)